src/V3LinkDot.cpp

```diff
@@ -986,6 +983,13 @@ class LinkDotFindVisitor final : public VNVisitor {
     }
     void visit(AstTypeTable*) override {}  // FindVisitor::
     void visit(AstConstPool*) override {}  // FindVisitor::
+    void visit(AstIfaceRefDType* nodep) override {
+        if (m_statep->forPrimary() && nodep->isVirtual() && nodep->ifacep()
```

Comment:
Though likely doesn't matter given how this visitor currently works, is better future proofing.

```suggestion
    void visit(AstIfaceRefDType* nodep) override {
        iterateChildren(nodep);
        if (m_statep->forPrimary() && nodep->isVirtual() && nodep->ifacep()
```